src/controlnet_aux/hed/__init__.py

```diff
@@ -78,7 +79,11 @@ def to(self, device):
         self.netNetwork.to(device)
         return self
     
-    def __call__(self, input_image, detect_resolution=512, image_resolution=512, safe=False, return_pil=True, scribble=False):
+    def __call__(self, input_image, detect_resolution=512, image_resolution=512, safe=False, return_pil=None, scribble=False, output_type="pil"):
```

Comment:
I'd also add a kwarg here for the same reason as described here: https://github.com/patrickvonplaten/controlnet_aux/pull/49/files#r1214565652